string detach_model
string detach_joint
---
bool status